wagtail/admin/views/pages/listing.py

```diff
@@ -218,53 +218,58 @@ def get_queryset(self):
             "content_type", "sites_rooted_here"
         ) & self.permission_policy.explorable_instances(self.request.user)
 
-        pages = self.filter_queryset(pages)
-
-        self.ordering = self.get_ordering()
-
-        if not self.is_searching:
-            if self.ordering == "ord":
-                # preserve the native ordering from get_children()
-                pass
-            elif self.ordering == "latest_revision_created_at":
-                # order by oldest revision first.
-                # Special case NULL entries - these should go at the top of the list.
-                # Do this by annotating with Count('latest_revision_created_at'),
-                # which returns 0 for these
-                pages = pages.annotate(
-                    null_position=Count("latest_revision_created_at")
-                ).order_by("null_position", "latest_revision_created_at")
-            elif self.ordering == "-latest_revision_created_at":
-                # order by oldest revision first.
-                # Special case NULL entries - these should go at the end of the list.
-                pages = pages.annotate(
-                    null_position=Count("latest_revision_created_at")
-                ).order_by("-null_position", "-latest_revision_created_at")
-            else:
-                pages = pages.order_by(self.ordering)
-
         # We want specific page instances, but do not need streamfield values here
         pages = pages.defer_streamfields().specific()
 
-        # allow hooks defer_streamfieldsyset
-        for hook in hooks.get_hooks("construct_explorer_page_queryset"):
-            pages = hook(self.parent_page, pages, self.request)
-
         # Annotate queryset with various states to be used later for performance optimisations
         if getattr(settings, "WAGTAIL_WORKFLOW_ENABLED", True):
             pages = pages.prefetch_workflow_states()
 
         pages = pages.annotate_site_root_state().annotate_approved_schedule()
```

Comment:
I explored the idea of adding `annotate_queryset` in 86a10b85eee0754327b0cd9ab4ae9919c318ad7a and 7ec21f566e92d453a74e61a1e4ac2a78927007cc, but I'm not sure if grouping the annotations in one method is better than putting them where they're actually needed (close to the logic that uses them, e.g. annotating `_updated_at` in `order_queryset`).